mod_reforged/hooks/config/strings.nut

```diff
@@ -656,9 +656,8 @@ local vanillaDescriptions = [
 				Type = ::UPD.EffectType.Passive,
 				Description = [
 					"Skills build up " + ::MSU.Text.colorPositive("25%") + " less [Fatigue.|Concept.Fatigue]",
-					"The [Action Point|Concept.ActionPoints] cost of [Spearwall|Skill+spearwall] is reduced by " + ::MSU.Text.colorPositive("1") + ".",
```

Comment:
This is not mentioned in the commit message. Was this put in by accident?
If not, please give me few words to understand this change. And ideally this change should be part of the commit message

Reply:
That got included in that by mistake. I've pulled it out into a separate commit.